<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Broadcast chat</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="nickTextBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>161</width>
      <height>27</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="nickLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Nickname:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="joinPushButton">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>101</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Join</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="chatTextBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>541</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="sendTextBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>290</y>
      <width>541</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="sendPushButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>350</y>
      <width>98</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Send</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>564</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
